modules/dnn/src/cuda4dnn/init.hpp

```diff
@@ -87,9 +91,7 @@ namespace cv { namespace dnn { namespace cuda4dnn {
         CUDA4DNN_CHECK_CUDA(cudaDeviceGetAttribute(&minor, cudaDevAttrComputeCapabilityMinor, device_id));
 
         int version = major * 10 + minor;
-        if (version < 53)
-            return false;
-        return true;
+        return (version >= 53);
```

Comment:
Taking into account that the `version` is an integer variable, I think it would be a bit cleaner if we use `> 52` for this check.

```suggestion
        return (version > 52);
```

Reply:
I prefer to stay with 53. It's well known constant in CUDA world. Also it's easier to "grep".